### DISCUSSION TITLE
Which version of skywalking (oap) compatible with elasticsearch ( version 8.11.3,  version 8.8.0,   version 8.3.0 )

### DISCUSSION THREAD

**Discussion by DeBruyne2020** [apache/skywalking#12013] [Q&A]
I have a question for help , 🙏
Which version of skywalking (oap) compatible with elasticsearch ( version 8.11.3,   version 8.8.0,   version 8.3.0 )  ?

🙏

**Comment by wu-sheng** [comment]
They should be same, and supported by the latest release.

We didn't test them one by one.

**Reply by DeBruyne2020** [reply]
"the latest release",  do you mean the version  9.7.0  ？

**Reply by wu-sheng** [reply]
Yes, we just require basic features of elasticsearch. 
Do you face some issues? Why are you asking for those specific versions?

**Reply by DeBruyne2020** [reply]
------ "Do you face some issues? Why are you asking for those specific versions?"
I need to solve the CVE-2022-1471 about snakeyaml  for elasticsearch.  As I know,  the elasticsearch which >= 8.3.0 version  have solved the CVE  and  Now thers is only version-8.11.3 elasticsearch  in our company-software-library which version >=8.3.0,  so We decide to use the version-8.11.3 elasticsearch ,  so We need to ensure our skywalking can work well (compitible with elasticsearch version 8.11.3 ) .   
 If the version-8.11.3 elasticsearch cannot satisfy our need,  then we may consider import the version 8.3.0 or 8.8.0 elasticsearch to our company-software-library.

**Reply by wu-sheng** [reply]
I think it should be fine. You could test, if there is an issue, let's know.

**Reply by DeBruyne2020** [reply]
Ok, we 'll do test.

Thanks a lot 🙏

**Reply by krishnaprakashthokala** [reply]
Hi , just i would like to know that have you installed without issues with  helm chart , is it OAP server running ?

**Comment by krishnaprakashthokala** [comment]
> ------ "Do you face some issues? Why are you asking for those specific versions?" I need to solve the [CVE-2022-1471](https://github.com/advisories/GHSA-mjmj-j48q-9wg2) about snakeyaml for elasticsearch. As I know, the elasticsearch which >= 8.3.0 version have solved the CVE and Now thers is only version-8.11.3 elasticsearch in our company-software-library which version >=8.3.0, so We decide to use the version-8.11.3 elasticsearch , so We need to ensure our skywalking can work well (compitible with elasticsearch version 8.11.3 ) . If the version-8.11.3 elasticsearch cannot satisfy our need, then we may consider import the version 8.3.0 or 8.8.0 elasticsearch to our company-software-library.
